pkg/ddevapp/global_dotddev_assets/commands/web/craft

```diff
@@ -3,11 +3,15 @@
 #ddev-generated
 ## Description: Run Craft CMS command inside the web container
 ## Usage: craft [flags] [args]
-## Example: "ddev craft db/backup" or "craft db/backup ./my-backups" (see https://craftcms.com/docs/4.x/console-commands.html)
-## ProjectTypes: craftcms
+## Example: "ddev craft db/backup" or "ddev craft db/backup ./my-backups" (see https://craftcms.com/docs/4.x/console-commands.html)
+## ProjectTypes: craftcms,php
 ## ExecRaw: true
 
-CRAFT_CMD_ROOT=${CRAFT_CMD_ROOT:="./"}
-
-cd "${CRAFT_CMD_ROOT}"
-php ./craft "$@"
+if grep -q "type: php" ./.ddev/config.yaml; then
+    echo "The craft command is only available in the craftcms project type. You can update this in your project's config file, followed by restarting the DDEV project."
+else
+    CRAFT_CMD_ROOT=${CRAFT_CMD_ROOT:="./"}
+    
+    cd "${CRAFT_CMD_ROOT}"
+    php ./craft "$@"
```

Comment:
Just a note that `php ./craft` is pretty much redundant and `php craft` is the same. But that's not important. I do keep trying to understand if craft may be somewhere else, in which case we could consider using `## HostWorkingDir: true` to run it in the same relative directory they're in. I don't think there's any reason to fiddle with this right now, jut talking out loud.

Reply:
Yeah the check for the environment variable is likely better, I agree. I just didn't know it existed.

Reply:
Updated, thanks for the review!